github-profiles/Marcos.-0222/style.css

```diff
@@ -0,0 +1,106 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:300,400,500';
+
+header a {
+    border: 1px solid #e81c4f;
+    border-radius: 290486px;
+    color: white;
+    font-size: 0.6em;
+    letter-spacing: 0.2em;
+    padding: 1em 2em;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: none 200ms ease-out;
+    transition-property: color, background;
+  }
+  
+  header a:hover {
+    background:  #e81c4f;
+    color: white;
+  }
+body {
+  font-family: "Roboto", "Helvetica", "Arial", sans-serif;
+}
+body {
+    margin: 0 auto;
+    max-width: 50em;
+  }
+  body {
+    line-height: 1.5;
+    padding: 4em 1em;
+  }
+  
+  h2 {
+    margin-top: 1em;
+    padding-top: 1em;
+  }
+  body {
+    background-color: rgb(60, 45, 196);
+    color: rgb(255, 255, 255);
+  }
+  
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  strong {
+    color: #00095c;
+  }
+  a {
+    color:rgb(43, 226, 43)
+  }
+  code,
+pre {
+  background: #eee;
+}
+
+code {
+  padding: 2px 4px;
+  vertical-align: text-bottom;
+}
+
+pre {
+  padding: 1em;
+}
+body {
+    color: #fbff00;
+  }
+  
+  code,
+  pre {
+    background: #f5f7f9;
+    border-bottom: 1px solid #d8dee9;
+    color: #ffffff;
+  }
+  
+  pre {
+    border-left: 2px solid #69c;
+  }
+  header {
+    background-color: #263d36;
+    background-image: url("header.jpg");
+    background-position: center top;
+    background-repeat: no-repeat;
+    background-size: cover;
+    line-height: 1.2;
+    padding: 10vw 2em;
+    text-align: center;
+  }
+  body img {
```

Comment:
I don't know why this comment wasn't showed on the previous review, but, you shouldn't add style to an image on your `.css` file when you are not using any image on your `.html`.